### ISSUE TITLE
avoid-unused-parameters does not report parameters from typedef

### ISSUE THREAD

**Issue by DenisBogatirov** [solid-software/solid_lints#58]
Rule should report positional parameters from typedef if their name are not underscores.

Code below represents expected behavior.

```
typedef MaxFun = int Function(int a, int b);

// expect_lint: avoid-unused-parameters
final MaxFun maxFunLint = (int a, int b) => 1;
  
final MaxFun maxFunOk = (int _, int __) => 1;
```
